winter/core/annotation_decorator.py

```diff
@@ -0,0 +1,47 @@
+import inspect
+import types
+import typing
+
+from .component import Component
+from .component_method import ComponentMethod
+
+
+def annotate(annotation: typing.Any) -> typing.Union[typing.Type, ComponentMethod]:
+
+    def wrapper(func_or_cls: typing.Union[typing.Type, types.FunctionType, ComponentMethod]):
+
```

Comment:
Please remove these blank lines: 
_There's no blank line either before or after the docstring_
https://www.python.org/dev/peps/pep-0257/